### PR TITLE
Removed hardcoded localhost url when using browsing and publisher messages for queues through UI

### PR DESCRIPTION
**Pull request by hemikak**
Browsing and publishing messages for queues did not work through the UI as the the amqp url was hardcoded with "localhost". This was changed to pick up from AndesConfiguration(bindAddress in broker.xml).

JIRA - https://wso2.org/jira/browse/MB-923

### CODE REVIEW

File: components/andes/org.wso2.carbon.andes.core/src/main/java/org/wso2/carbon/andes/core/internal/util/Utils.java [hunk 1]
```diff
@@ -388,73 +441,74 @@ public static String getMsgContentType(Message queueMessage) {
...
-            while (mapEnu.hasMoreElements()) {
-                String mapName = (String) mapEnu.nextElement();
-                String mapVal = mapMessage.getObject(mapName).toString();
-                wholeMsg = StringEscapeUtils.escapeHtml(sb.append(mapName).append(": ")
-                        .append(mapVal).append(", ").toString()).trim();
+        if (message != null) {
+            if (message instanceof TextMessage) {
+                String textMessage = ((TextMessage) message).getText();
+                wholeMsg = StringEscapeUtils.escapeHtml(textMessage).trim();
+                if (wholeMsg.length() >= 15) {
```
Comment (akalankapagoda) [inline components/andes/org.wso2.carbon.andes.core/src/main/java/org/wso2/carbon/andes/core/internal/util/Utils.java]: The number 15 in this method seems like a magic number. If it is a constant value can we extract it as a constant an explain what it is?

File: components/andes/org.wso2.carbon.andes.core/src/main/java/org/wso2/carbon/andes/core/internal/util/Utils.java [hunk 2]
```diff
@@ -22,46 +22,55 @@
...
-import org.wso2.carbon.andes.core.types.Queue;
-import org.wso2.carbon.utils.ServerConstants;
-
-import javax.jms.*;
-import javax.xml.namespace.QName;
-import javax.xml.stream.XMLStreamException;
-
-
+/**
+ * Provides
```
Comment (ramith) [inline components/andes/org.wso2.carbon.andes.core/src/main/java/org/wso2/carbon/andes/core/internal/util/Utils.java]: Provides what? !!! :D

File: components/andes/org.wso2.carbon.andes.core/src/main/java/org/wso2/carbon/andes/core/internal/util/Utils.java (hunk 1 above)
Comment (ramith) [inline components/andes/org.wso2.carbon.andes.core/src/main/java/org/wso2/carbon/andes/core/internal/util/Utils.java]: +1

Comment (hemikak) [inline components/andes/org.wso2.carbon.andes.core/src/main/java/org/wso2/carbon/andes/core/internal/util/Utils.java]: True, fixed in https://github.com/wso2/carbon-business-messaging/pull/30

File: components/andes/org.wso2.carbon.andes.core/src/main/java/org/wso2/carbon/andes/core/internal/util/Utils.java (hunk 2 above)
Comment (hemikak) [inline components/andes/org.wso2.carbon.andes.core/src/main/java/org/wso2/carbon/andes/core/internal/util/Utils.java]: Oops. Left out by mistake. Fixed in https://github.com/wso2/carbon-business-messaging/pull/30

**Pull request merged**